lean/components/docker/lean_runner.py

```diff
@@ -183,7 +183,7 @@ def get_basic_docker_config(self,
         # Install the required modules when they're needed
         if lean_config.get("data-provider", None) == "QuantConnect.Lean.Engine.DataFeeds.DownloaderDataProvider" \
             and lean_config.get("data-downloader", None) == "TerminalLinkDataDownloader":
-            self._module_manager.install_module(TERMINAL_LINK_PRODUCT_ID, lean_config["job-organization-id"])
+            self._module_manager.install_module(TERMINAL_LINK_PRODUCT_ID, lean_config["organization-id"])
```

Comment:
`job-organization-id` is what lean config.json is using, see https://github.com/QuantConnect/Lean/blob/master/Launcher/config.json#L71 , which is != than what the lean cli lean.json has, which will now be `organization-id` ?

Reply:
Right, fixing